app/models/concerns/scripts/import.rb

```diff
@@ -47,8 +47,9 @@ def import_scripts scripts_data
       def import_script data, scripts_data
         return if data == :imported
 
-        uuid   = data['uuid']
-        script = find_by uuid: uuid
+        uuid           = data['uuid']
+        script         = find_by uuid: uuid
+        script_version = data.delete('current_version')
```

Comment:
```suggestion
        version = data.delete 'current_version'
```
Acá si para la variable le dejaría solo `version`.

Reply:
dale